README.md

```diff
@@ -107,11 +107,22 @@ If you are using the provided workspace.json, use Weather Company Data to provid
 Follow this link and hit `Create`:
 * [**Weather Company Data**](https://console.ng.bluemix.net/catalog/services/weather-company-data)
 
-This service includes an OpenWhisk package. Run the following command to update your OpenWhisk bindings:
+This service includes an OpenWhisk package.
+Run the following to [install the OpenWhisk bindings for IBM cloud](https://console.bluemix.net/openwhisk/learn/cli):
+```
+$ bx plugin install Cloud-Functions -r Bluemix
+```
+
+Run the following command to update your OpenWhisk bindings if they are already installed:
 ```
 $ bx wsk package refresh
 ```
 
+Run the following to test OpenWhisk on IBM Cloud:
+```
+$ bx wsk action invoke /whisk.system/utils/echo -p message hello --result
+Copy
```

Comment:
Is the line "Copy" here just a copy/paste error?

Reply:
yes

Reply:
merge then fix